kouncil-frontend/libs/common-login/src/lib/login/user.ts

```diff
@@ -0,0 +1,9 @@
+export class User {
```

Comment:
do you really need class?
see interface vs class in ts https://stackoverflow.com/a/51716899